main.go

```diff
@@ -62,10 +64,12 @@ func main() {
 		// Deprecated: use defaultMaxConcurrentReconciles
 		defaultMaxWorkers int
 	)
-
+	//todo(camilamacedo86): why the port is not 8080?
 	pflag.StringVar(&metricsAddr, "metrics-addr", "0.0.0.0:8383", "The address the metric endpoint binds to.")
```

Comment:
why the port is not 8080?

Reply:
The scaffolds will work with the default port. I think to keep it sample we should change that. See: https://github.com/kubernetes-sigs/kubebuilder/search?q=8080&unscoped_q=8080

Reply:
Addressed in https://github.com/joelanford/helm-operator/issues/19 to be easier discuss.